components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -66,31 +69,34 @@
 
     public static final Integer charactersPerLine = 100;
 
+    private TranslationManager translationManager;
+
+    public DataToCellConverter()
+    {
+        try {
+            this.translationManager = ComponentManagerRegistry.getContextComponentManager().getInstance(
+                TranslationManager.class);
+        } catch (ComponentLookupException ex) {
+            System.out.println(ex.toString());
+        }
+
+    }
+
     public void phenotypeSetup(Set<String> enabledFields) throws Exception
     {
         String sectionName = "phenotype";
         String[] fieldIds = { "phenotype", "phenotype_code", "phenotype_combined", "phenotype_code_meta",
-            "phenotype_meta", "negative_phenotype", "negative_phenotype_code", "negative_phenotype_combined",
-            "phenotype_by_section" };
+        "phenotype_meta", "negative_phenotype", "negative_phenotype_code", "negative_phenotype_combined",
+        "phenotype_by_section" };
```

Comment:
Irrelevant whitespace changes. Should be committed straight to master or as a separate PR.